legacy/web/src/test/java/org/jboss/as/web/test/WebMigrateTestCase.java

```diff
@@ -302,6 +303,10 @@ protected RunningMode getRunningMode() {
             return RunningMode.ADMIN_ONLY;
         }
 
+        @Override
+        protected ProcessType getProcessType() {
+            return ProcessType.SELF_CONTAINED;
```

Comment:
changing it to host controller means it will do a domain mode migration, which would mean the test has to change.